Tensile/KernelWriterAssembly.py

```diff
@@ -2064,72 +2064,71 @@ def functionSignature(self, kernel ):
+      kStr += self.v2Argument(                               'A',     '8',      '8', "GlobalBuffer", srcValueType, "Generic")
+      kStr += self.v2Argument(                               'B',     '8',      '8', "GlobalBuffer", srcValueType, "Generic")
 
       if kernel["ProblemType"]["DataType"].isHalf() or \
          kernel["ProblemType"]["DataType"].isSingle() or \
          kernel["ProblemType"]["DataType"].isInt8x4():
-        kStr += self.v2Argument(                           "alpha",     '4',      '4',      "ByValue", cptValueType)
+        kStr += self.v2Argument(                         "alpha",     '4',      '4',      "ByValue", cptValueType)
       elif kernel["ProblemType"]["DataType"].isDouble():
-        kStr += self.v2Argument(                           "alpha", cptSize, cptAlign,      "ByValue", cptValueType)
+        kStr += self.v2Argument(                         "alpha", cptSize, cptAlign,      "ByValue", cptValueType)
 
       if kernel["ProblemType"]["UseBeta"]:
         if kernel["ProblemType"]["DataType"].isHalf() or \
            kernel["ProblemType"]["DataType"].isSingle() or \
            kernel["ProblemType"]["DataType"].isInt8x4():
-          kStr += self.v2Argument(                          "beta",     '4',      '4',      "ByValue", cptValueType)
+          kStr += self.v2Argument(                        "beta",     '4',      '4',      "ByValue", cptValueType)
       elif kernel["ProblemType"]["DataType"].isDouble():
-        kStr += self.v2Argument(                            "beta", cptSize, cptAlign,      "ByValue", cptValueType)
+        kStr += self.v2Argument(                          "beta", cptSize, cptAlign,      "ByValue", cptValueType)
 
       for i in range(0, self.numSgprStridesD):
-        kStr += self.v2Argument(                     "strideD%u"%i,     '4',      '4',      "ByValue",        "U32")
+        kStr += self.v2Argument(                   "strideD%u"%i,     '4',      '4',      "ByValue",        "U32")
 
       for i in range(0, self.numSgprStridesC):
-        kStr += self.v2Argument(                     "strideC%u"%i,     '4',      '4',      "ByValue",        "U32")
+        kStr += self.v2Argument(                   "strideC%u"%i,     '4',      '4',      "ByValue",        "U32")
 
       for i in range(0, self.numSgprStridesA):
-        kStr += self.v2Argument(                     "strideA%u"%i,     '4',      '4',      "ByValue",        "U32")
+        kStr += self.v2Argument(                   "strideA%u"%i,     '4',      '4',      "ByValue",        "U32")
 
       for i in range(0, self.numSgprStridesB):
-        kStr += self.v2Argument(                     "strideB%u"%i,     '4',      '4',      "ByValue",        "U32")
+        kStr += self.v2Argument(                   "strideB%u"%i,     '4',      '4',      "ByValue",        "U32")
 
 
       for i in range(0, self.numSgprSizesFree):
```

Comment:
this does not match the actual kernel args for
Cijk_Ailk_Bjlk_SB_MT64x128x16_SE_APM1_AF0EM1_AF1EM1_AMAS3_ASEM1_BL1_DTL0_EPS1_FL0_GRVW4_GSU1_ISA906_IU1_K1_KLA_LPA0_LPB0_LDL1_MGWVW1_NLCA1_NLCB1_PK0_PGR1_PLR1_RK0_SU32_SNLL0_TT4_8_USFGRO0_VAW1_VW4_WG16_16_1_WGM1

  struct {
    // Size of Tensor's packed dims, in elements
    uint64_t tensor2dSizeC;
    uint64_t tensor2dSizeA;
    uint64_t tensor2dSizeB;
    float * dataD;
    const float * dataC;
    const float * dataA;
    const float * dataB;
    float alpha;
    float beta;
    unsigned int strideD1J;
    unsigned int strideD2K;
    unsigned int strideC1J;
    unsigned int strideC2K;
    unsigned int strideA1L;
    unsigned int strideA2K;
    unsigned int strideB1L;
    unsigned int strideB2K;
    unsigned int sizeI;
    unsigned int sizeJ;
    unsigned int sizeK;
    unsigned int sizeL;
    int staggerUIter;
    unsigned int problemNumGroupTiles0;
    unsigned int problemNumGroupTiles1;
    unsigned int magicNumberProblemNumGroupTiles0;
    unsigned int gridNumWorkGroups0;
    unsigned int numFullBlocks;
    unsigned int wgmRemainder1;
    unsigned int magicNumberWgmRemainder1;
    unsigned int pad;
  } hipFunctionArgs;